tests/XRootD/http.sh

```diff
@@ -189,4 +189,62 @@ function test_http() {
   receivedHeader=$(grep -i 'Test:' "$outputFilePath")
   assert_eq "1" "$(echo "$receivedHeader" | wc -l | sed 's/^ *//')" "Incorrect number of 'Test' header values"
   assert_eq "$expectedHeader" "$receivedHeader" "HEAD is missing statically-defined Test header"
+
+  set -ex
```

Comment:
Let's avoid setting `e` here at least; we should explicitly check for failures where needed (`e` can have bad interactions with traps).

`x` I'm indifferent on but likely should be cleaned up to have the unit test be less verbose.

Reply:
I agree, let's try not to make tests too verbose. A clear message on failure is more than enough. Tests that pass don't need to create any output.